### PR TITLE
Fix timestamps in chrome trace output

### PR DESCRIPTION
**Pull request by devongovett**

### CODE REVIEW

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.45s | +5.00ms |
| Cached | 475.00ms | -35.00ms 🚀 |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/modern/index.f81b5b4b.js | 1.11kb | +0.00b | 536.00ms | +45.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 538.00ms | +30.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 536.00ms | +61.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 47.00ms | -26.00ms 🚀 |
| dist/legacy/index.9ff7c523.js | 1.11kb | +0.00b | 48.00ms | -25.00ms 🚀 |
| dist/modern/index.f81b5b4b.js | 1.11kb | +0.00b | 65.00ms | -8.00ms 🚀 |
| dist/modern/index.html | 701.00b | +0.00b | 47.00ms | -26.00ms 🚀 |
| dist/modern/index.77ce15d4.css | 77.00b | +0.00b | 47.00ms | -26.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 17.57s | -116.00ms |
| Cached | 8.04s | +238.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.d85c0b32.js | 4.20kb | +0.00b | 835.00ms | +141.00ms ⚠️ |
| dist/UserProfile.854f4adc.js | 1.76kb | +0.00b | 835.00ms | +133.00ms ⚠️ |
| dist/NotFound.9195d8a0.js | 530.00b | +0.00b | 203.00ms | -54.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 470.00ms | -38.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 485.61kb | +0.00b | 63.00ms | -17.00ms 🚀 |
| dist/PermalinkedComment.d85c0b32.js | 4.20kb | +0.00b | 63.00ms | -17.00ms 🚀 |
| dist/UserProfile.854f4adc.js | 1.76kb | +0.00b | 63.00ms | -17.00ms 🚀 |
| dist/NotFound.9195d8a0.js | 530.00b | +0.00b | 42.00ms | -146.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 43.00ms | -25.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 1.92m | +3.18s |
| Cached | 2.77s | +194.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.61dfb46d.js | 2.47mb | +0.00b | 1.14m | +13.55s ⚠️ |
| dist/editorView.51048e78.js | 587.53kb | +0.00b | 30.59s | -35.21s 🚀 |
| dist/popup.4ddc9247.js | 169.25kb | +0.00b | 16.16s | +2.04s ⚠️ |
| dist/EmojiPickerComponent.00dbc417.js | 139.01kb | +0.00b | 9.36s | +1.27s ⚠️ |
| dist/Toolbar.b9000328.js | 99.77kb | +0.00b | 16.16s | +1.83s ⚠️ |
| dist/card.63e17a92.js | 54.02kb | +0.00b | 9.34s | -564.00ms 🚀 |
| dist/card.72b48e9e.js | 51.77kb | +0.00b | 16.05s | +3.87s ⚠️ |
| dist/Modal.00b0c761.js | 39.18kb | +0.00b | 3.51s | -235.00ms 🚀 |
| dist/component.86f6e486.js | 30.82kb | +0.00b | 3.33s | +1.57s ⚠️ |
| dist/esm.eab73ebc.js | 27.67kb | +0.00b | 6.20s | -881.00ms 🚀 |
| dist/smartMediaEditor.20d1e78e.js | 16.85kb | +0.00b | 11.61s | -818.00ms 🚀 |
| dist/js.0a808d22.js | 16.55kb | +0.00b | 3.49s | -251.00ms 🚀 |
| dist/ui.617a16c0.js | 14.21kb | +0.00b | 9.18s | -600.00ms 🚀 |
| dist/workerHasher.83104f73.js | 11.90kb | +0.00b | 6.44s | -466.00ms 🚀 |
| dist/component.053770c1.js | 6.20kb | +0.00b | 3.51s | -237.00ms 🚀 |
| dist/media-viewer.ebf61ecb.js | 4.01kb | +0.00b | 9.66s | -689.00ms 🚀 |
| dist/png-chunks-extract.692b85d1.js | 3.55kb | +0.00b | 3.49s | -251.00ms 🚀 |
| dist/index.10f02a5e.css | 3.46kb | +0.00b | 16.64s | +997.00ms ⚠️ |
| dist/Modal.29e822bf.js | 3.14kb | +0.00b | 1.02s | -2.04s 🚀 |
| dist/clipboard.7188d6f8.js | 2.97kb | +0.00b | 10.54s | -3.56s 🚀 |
| dist/16.41f46c2f.js | 2.49kb | +0.00b | 1.02s | -712.00ms 🚀 |
| dist/card.a2f58842.js | 2.15kb | +0.00b | 9.94s | -535.00ms 🚀 |
| dist/date.5e8b4b49.js | 1.96kb | +0.00b | 3.94s | -217.00ms 🚀 |
| dist/16.8ab9c9cf.js | 1.86kb | +0.00b | 4.13s | +237.00ms ⚠️ |
| dist/browser.8d5afc2b.js | 1.82kb | +0.00b | 10.55s | -1.45s 🚀 |
| dist/16.aa2b1e8c.js | 1.79kb | +0.00b | 3.73s | +194.00ms ⚠️ |
| dist/workerHasher.5caa41b8.js | 1.75kb | +0.00b | 6.44s | -336.00ms 🚀 |
| dist/workerHasher.087b011b.js | 1.75kb | +0.00b | 11.43s | -2.69s 🚀 |
| dist/list-number.cc386df7.js | 1.68kb | +0.00b | 4.03s | -555.00ms 🚀 |
| dist/code.be6c45b5.js | 1.61kb | +0.00b | 3.94s | -488.00ms 🚀 |
| dist/link.f7c12162.js | 1.53kb | +0.00b | 4.03s | -555.00ms 🚀 |
| dist/heading6.a29cffb2.js | 1.53kb | +0.00b | 7.66s | +2.64s ⚠️ |
| dist/heading3.2326b3c4.js | 1.51kb | +0.00b | 5.27s | +807.00ms ⚠️ |
| dist/16.b19b33ca.js | 1.51kb | +0.00b | 3.70s | -443.00ms 🚀 |
| dist/16.fe7e09db.js | 1.46kb | +0.00b | 3.70s | -443.00ms 🚀 |
| dist/emoji.c2ecc9d7.js | 1.45kb | +0.00b | 3.94s | -217.00ms 🚀 |
| dist/16.7fc8af6d.js | 1.45kb | +0.00b | 1.02s | -712.00ms 🚀 |
| dist/heading5.fa40b79e.js | 1.40kb | +0.00b | 5.34s | -1.41s 🚀 |
| dist/16.c4941d0d.js | 1.36kb | +0.00b | 4.13s | +229.00ms ⚠️ |
| dist/16.a4fa1a69.js | 1.34kb | +0.00b | 1.02s | -712.00ms 🚀 |
| dist/heading2.fa6667ad.js | 1.33kb | +0.00b | 4.21s | -260.00ms 🚀 |
| dist/mention.539104d0.js | 1.29kb | +0.00b | 4.03s | -251.00ms 🚀 |
| dist/heading4.45b46e5b.js | 1.29kb | +0.00b | 5.27s | +802.00ms ⚠️ |
| dist/Modal.760be919.js | 1.28kb | +0.00b | 3.49s | -251.00ms 🚀 |
| dist/16.55b14beb.js | 1.27kb | +0.00b | 1.02s | -712.00ms 🚀 |
| dist/layout.61e66223.js | 1.27kb | +0.00b | 4.03s | -555.00ms 🚀 |
| dist/16.45835899.js | 1.27kb | +0.00b | 3.71s | -191.00ms 🚀 |
| dist/16.e83a0923.js | 1.26kb | +0.00b | 3.70s | -321.00ms 🚀 |
| dist/16.d315628f.js | 1.26kb | +0.00b | 3.70s | -443.00ms 🚀 |
| dist/divider.d62a9038.js | 1.25kb | +0.00b | 3.94s | -217.00ms 🚀 |
| dist/component.5df34eb8.js | 1.23kb | +0.00b | 3.28s | +216.00ms ⚠️ |
| dist/16.f1c68cda.js | 1.23kb | +0.00b | 3.28s | +216.00ms ⚠️ |
| dist/decision.aa61b855.js | 1.21kb | +0.00b | 3.94s | -217.00ms 🚀 |
| dist/panel-warning.96ab13ca.js | 1.21kb | +0.00b | 4.10s | -266.00ms 🚀 |
| dist/list.2f1b17d3.js | 1.18kb | +0.00b | 4.03s | -256.00ms 🚀 |
| dist/heading1.608781c0.js | 1.18kb | +0.00b | 4.20s | -271.00ms 🚀 |
| dist/panel.e409d6fe.js | 1.10kb | +0.00b | 4.12s | -251.00ms 🚀 |
| dist/table.d445bf7f.js | 1.09kb | +0.00b | 4.20s | -255.00ms 🚀 |
| dist/panel-success.26c394f0.js | 1.05kb | +0.00b | 4.10s | -246.00ms 🚀 |
| dist/media-card-analytics-error-boundary.ece1d621.js | 999.00b | +0.00b | 10.71s | -1.42s 🚀 |
| dist/media-viewer-analytics-error-boundary.9e64afc6.js | 995.00b | +0.00b | 9.68s | -710.00ms 🚀 |
| dist/simpleHasher.26a74474.js | 755.00b | +0.00b | 6.48s | -1.54s 🚀 |
| dist/simpleHasher.042b95c1.js | 755.00b | +0.00b | 11.47s | -2.65s 🚀 |
| dist/index.html | 119.00b | +0.00b | 2.55s | +794.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.48483ca4.js | 2.43mb | +0.00b | 570.00ms | +51.00ms ⚠️ |
| dist/pdfRenderer.6d316c6f.js | 1.11mb | +0.00b | 640.00ms | +75.00ms ⚠️ |
| dist/editorView.51048e78.js | 587.53kb | +0.00b | 658.00ms | +78.00ms ⚠️ |
| dist/popup.dbfaab2f.js | 169.17kb | +0.00b | 652.00ms | +78.00ms ⚠️ |
| dist/EmojiPickerComponent.00dbc417.js | 139.01kb | +0.00b | 631.00ms | +71.00ms ⚠️ |
| dist/Toolbar.3a17c008.js | 99.68kb | +0.00b | 658.00ms | +78.00ms ⚠️ |
| dist/media-viewer.f5985104.js | 72.78kb | +0.00b | 640.00ms | +75.00ms ⚠️ |
| dist/card.ee3301bd.js | 53.93kb | +0.00b | 634.00ms | +63.00ms ⚠️ |
| dist/card.b7089528.js | 51.68kb | +0.00b | 651.00ms | +60.00ms ⚠️ |
| dist/Modal.d1648f35.js | 39.01kb | +0.00b | 569.00ms | +51.00ms ⚠️ |
| dist/component.86f6e486.js | 30.82kb | +0.00b | 569.00ms | +51.00ms ⚠️ |
| dist/esm.eab73ebc.js | 27.67kb | +0.00b | 631.00ms | +71.00ms ⚠️ |
| dist/component.7d43a56a.js | 22.54kb | +0.00b | 559.00ms | +81.00ms ⚠️ |
| dist/DatePicker.28d662bb.js | 21.12kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/smartMediaEditor.2987dd07.js | 16.76kb | +0.00b | 658.00ms | +78.00ms ⚠️ |
| dist/js.0a808d22.js | 16.55kb | +0.00b | 558.00ms | +81.00ms ⚠️ |
| dist/dropzone.04d32b61.js | 16.00kb | +0.00b | 651.00ms | +60.00ms ⚠️ |
| dist/ui.617a16c0.js | 14.21kb | +0.00b | 634.00ms | +63.00ms ⚠️ |
| dist/workerHasher.83104f73.js | 11.90kb | +0.00b | 631.00ms | +71.00ms ⚠️ |
| dist/component.053770c1.js | 6.20kb | +0.00b | 559.00ms | +81.00ms ⚠️ |
| dist/card.241fc843.js | 5.71kb | +0.00b | 640.00ms | +75.00ms ⚠️ |
| dist/media-viewer.ebf61ecb.js | 4.01kb | +0.00b | 640.00ms | +68.00ms ⚠️ |
| dist/png-chunks-extract.692b85d1.js | 3.55kb | +0.00b | 559.00ms | +81.00ms ⚠️ |
| dist/EmojiPickerComponent.39b93682.js | 3.53kb | +0.00b | 634.00ms | +63.00ms ⚠️ |
| dist/index.10f02a5e.css | 3.46kb | +0.00b | 654.00ms | +72.00ms ⚠️ |
| dist/dropzone.840966d2.js | 3.44kb | +0.00b | 652.00ms | +78.00ms ⚠️ |
| dist/Modal.29e822bf.js | 3.14kb | +0.00b | 549.00ms | +86.00ms ⚠️ |
| dist/clipboard.7188d6f8.js | 2.97kb | +0.00b | 651.00ms | +60.00ms ⚠️ |
| dist/16.41f46c2f.js | 2.49kb | +0.00b | 548.00ms | +85.00ms ⚠️ |
| dist/ResourcedEmojiComponent.9812a721.js | 2.15kb | +0.00b | 634.00ms | +63.00ms ⚠️ |
| dist/card.a2f58842.js | 2.15kb | +0.00b | 652.00ms | +80.00ms ⚠️ |
| dist/date.5e8b4b49.js | 1.96kb | +0.00b | 586.00ms | +68.00ms ⚠️ |
| dist/images.e479dcbe.js | 1.90kb | +0.00b | 591.00ms | +68.00ms ⚠️ |
| dist/feedback.00d01eca.js | 1.86kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/16.8ab9c9cf.js | 1.86kb | +0.00b | 577.00ms | +67.00ms ⚠️ |
| dist/browser.8d5afc2b.js | 1.82kb | +0.00b | 651.00ms | +60.00ms ⚠️ |
| dist/16.aa2b1e8c.js | 1.79kb | +0.00b | 568.00ms | +77.00ms ⚠️ |
| dist/workerHasher.5caa41b8.js | 1.75kb | +0.00b | 631.00ms | +71.00ms ⚠️ |
| dist/workerHasher.087b011b.js | 1.75kb | +0.00b | 657.00ms | +83.00ms ⚠️ |
| dist/list-number.cc386df7.js | 1.68kb | +0.00b | 591.00ms | +68.00ms ⚠️ |
| dist/status.008ba29f.js | 1.68kb | +0.00b | 610.00ms | +68.00ms ⚠️ |
| dist/code.be6c45b5.js | 1.61kb | +0.00b | 586.00ms | +68.00ms ⚠️ |
| dist/link.f7c12162.js | 1.53kb | +0.00b | 602.00ms | +79.00ms ⚠️ |
| dist/heading6.a29cffb2.js | 1.53kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/heading3.2326b3c4.js | 1.51kb | +0.00b | 627.00ms | +73.00ms ⚠️ |
| dist/16.b19b33ca.js | 1.51kb | +0.00b | 576.00ms | +74.00ms ⚠️ |
| dist/16.15d9f682.js | 1.46kb | +0.00b | 568.00ms | +77.00ms ⚠️ |
| dist/16.fe7e09db.js | 1.46kb | +0.00b | 568.00ms | +66.00ms ⚠️ |
| dist/emoji.c2ecc9d7.js | 1.45kb | +0.00b | 591.00ms | +68.00ms ⚠️ |
| dist/16.7fc8af6d.js | 1.45kb | +0.00b | 548.00ms | +85.00ms ⚠️ |
| dist/16.474c8276.js | 1.44kb | +0.00b | 577.00ms | +59.00ms ⚠️ |
| dist/16.d49a0bb7.js | 1.40kb | +0.00b | 577.00ms | +67.00ms ⚠️ |
| dist/heading5.fa40b79e.js | 1.40kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/expand.944ff7ce.js | 1.38kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/16.c4941d0d.js | 1.36kb | +0.00b | 577.00ms | +67.00ms ⚠️ |
| dist/16.a4fa1a69.js | 1.34kb | +0.00b | 548.00ms | +85.00ms ⚠️ |
| dist/heading2.fa6667ad.js | 1.33kb | +0.00b | 611.00ms | +57.00ms ⚠️ |
| dist/16.ad29438e.js | 1.32kb | +0.00b | 577.00ms | +67.00ms ⚠️ |
| dist/16.3d7a389d.js | 1.31kb | +0.00b | 586.00ms | +68.00ms ⚠️ |
| dist/mention.539104d0.js | 1.29kb | +0.00b | 591.00ms | +61.00ms ⚠️ |
| dist/heading4.45b46e5b.js | 1.29kb | +0.00b | 627.00ms | +67.00ms ⚠️ |
| dist/Modal.760be919.js | 1.28kb | +0.00b | 558.00ms | +81.00ms ⚠️ |
| dist/16.55b14beb.js | 1.27kb | +0.00b | 548.00ms | +85.00ms ⚠️ |
| dist/layout.61e66223.js | 1.27kb | +0.00b | 602.00ms | +79.00ms ⚠️ |
| dist/16.45835899.js | 1.27kb | +0.00b | 577.00ms | +75.00ms ⚠️ |
| dist/16.e83a0923.js | 1.26kb | +0.00b | 568.00ms | +77.00ms ⚠️ |
| dist/16.d315628f.js | 1.26kb | +0.00b | 576.00ms | +74.00ms ⚠️ |
| dist/16.cc8ae77e.js | 1.26kb | +0.00b | 577.00ms | +75.00ms ⚠️ |
| dist/16.71387abf.js | 1.26kb | +0.00b | 567.00ms | +77.00ms ⚠️ |
| dist/divider.d62a9038.js | 1.25kb | +0.00b | 590.00ms | +67.00ms ⚠️ |
| dist/quote.05e17876.js | 1.25kb | +0.00b | 610.00ms | +72.00ms ⚠️ |
| dist/component.5df34eb8.js | 1.23kb | +0.00b | 570.00ms | +51.00ms ⚠️ |
| dist/action.7545becf.js | 1.23kb | +0.00b | 586.00ms | +68.00ms ⚠️ |
| dist/16.f1c68cda.js | 1.23kb | +0.00b | 569.00ms | +50.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.d5ccfa87.js | 1.23kb | +0.00b | 640.00ms | +75.00ms ⚠️ |
| dist/decision.aa61b855.js | 1.21kb | +0.00b | 586.00ms | +63.00ms ⚠️ |
| dist/panel-warning.96ab13ca.js | 1.21kb | +0.00b | 627.00ms | +89.00ms ⚠️ |
| dist/16.8cd85281.js | 1.18kb | +0.00b | 577.00ms | +67.00ms ⚠️ |
| dist/list.2f1b17d3.js | 1.18kb | +0.00b | 591.00ms | +68.00ms ⚠️ |
| dist/heading1.608781c0.js | 1.18kb | +0.00b | 611.00ms | +68.00ms ⚠️ |
| dist/panel-error.aa8fc53d.js | 1.11kb | +0.00b | 592.00ms | +61.00ms ⚠️ |
| dist/panel.e409d6fe.js | 1.10kb | +0.00b | 627.00ms | +89.00ms ⚠️ |
| dist/table.d445bf7f.js | 1.09kb | +0.00b | 611.00ms | +68.00ms ⚠️ |
| dist/panel-success.26c394f0.js | 1.05kb | +0.00b | 611.00ms | +73.00ms ⚠️ |
| dist/panel-note.0319a3fe.js | 1.05kb | +0.00b | 591.00ms | +53.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.3e4d365d.js | 1003.00b | +0.00b | 652.00ms | +62.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.2e3e2b28.js | 999.00b | +0.00b | 634.00ms | +69.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.ece1d621.js | 999.00b | +0.00b | 651.00ms | +77.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.9e64afc6.js | 995.00b | +0.00b | 652.00ms | +80.00ms ⚠️ |
| dist/simpleHasher.26a74474.js | 755.00b | +0.00b | 631.00ms | +60.00ms ⚠️ |
| dist/simpleHasher.042b95c1.js | 755.00b | +0.00b | 657.00ms | +83.00ms ⚠️ |

</p></details>

<details><summary>Three.js ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 12.84s | +410.00ms |
| Cached | 573.00ms | +40.00ms ⚠️ |

#### Cold Bundles

*No bundle changes detected.*

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/Three.js | 580.59kb | +0.00b | 102.00ms | +13.00ms ⚠️ |

</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/287713633230651916)**

File: packages/core/workers/src/Trace.js [hunk 1]
```diff
@@ -12,17 +12,17 @@ export default class Trace {
...
     this.eventId = 0;
-    this.init();
   }
 
   getEventId(): number {
     return this.eventId++;
   }
 
-  init() {
+  init(ts) {
```
Comment (wbinnssmith) [inline packages/core/workers/src/Trace.js:21]: looks like Flow wants an annotation here

Comment (mischnic) [reply packages/core/workers/src/Trace.js:21]:
```suggestion
  init(ts: number) {
```

**Review by mischnic — APPROVED** [review]

File: packages/core/workers/src/Trace.js [hunk 2]
```diff
@@ -52,6 +53,10 @@ export default class Trace {
   }
 
   addCPUProfile(name: string, profile: Profile) {
+    if (this.eventId === 0) {
+      this.init(profile.startTime);
+    }
+
```
Comment (mischnic) [inline packages/core/workers/src/Trace.js:59]:
```suggestion

```

**Pull request merged**